seventh_ex.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def swap_first_and_last_rows(randomized_array) -> None:
+    """
+    This function gets an array and swaps it's first rows with it's last rows.
+    :param randomized_array: the array that we will swap it's first and last rows.
+    """
+    swapped_array = randomized_array[::-1]
+
+
+def randomize_array() -> None:
```

Comment:
Again why aren;t you returning anything? Furthermore, each function should have one goal.